app/components/player/player.tsx

```diff
@@ -143,14 +150,23 @@ export function PlayerScores({ roomId, userId }: RoomProps) {
           />
         )
       ) : null}
-      {sortedOtherPlayers.map((p, i) => (
-        <PlayerScore
-          key={i}
-          player={p}
-          hasBoardControl={p.userId === boardControl}
-          winning={p.score === maxScore}
-        />
-      ))}
+      {sortedOtherPlayers.map((p, i) =>
+        canKick ? (
+          <KickPlayerForm
+            key={i}
+            roomId={roomId}
+            player={p}
+            winning={p.score === maxScore}
```

Comment:
alphabetize these and the fields in `PlayerScore`? that would make it easier to compare their props